.golangci.yml

```diff
@@ -5,6 +5,12 @@ run:
     - vendor
   skip-dirs-use-default: true
   modules-download-mode: vendor
+  build-tags:
+    - "aro"
+    - "containers_image_openpgp"
+    - "exclude_graphdriver_devicemapper"
+    - "exclude_graphdriver_btrfs"
```

Comment:
does this have a reason to be in this PR? or is it just some extra change bundled in it?

Reply:
Podman's bindings vendors in more of the image lib that causes build dep side effects.